src/main/kotlin/com/hibob/project/service/feedbackService.kt

```diff
@@ -15,7 +17,29 @@ class FeedbackService @Autowired constructor(private val feedbackDao: FeedbackDa
         if (validateFeedback(feedbackCreationRequest)) { feedbackDao.feedbackSubmission(feedbackCreationRequest, employee) }
     }
 
-    fun validateFeedback(feedbackCreationRequest: FeedbackCreationRequest): Boolean {
+    fun markFeedbackStatus(feedbackId: Long, status: StatusType, companyId: Long) {
+        val feedback = feedbackDao.getAllFeedbacks(companyId).find { it.id == feedbackId }
+            ?: throw NotFoundException("Feedback not found")
+
+        if (!feedback.isAnonymous) {
+            feedbackDao.markFeedbackStatus(feedback, status)
```

Comment:
better to do it the opposite:
if () throw

and then you don't even need the else block, just the code as is